Fix Regression in Full History CSV Percentiles

TaylorSMarks opened the pull request:
The docstring for _stats_history_data_rows says it should be outputting `*current* stats` (emphasis not added by me). In the original commit from December 3rd, 2019, it was properly using the `get_current_response_time_percentile` function.

On August 18th, 2020, the code was all reformatted and it looks like a regression was accidentally introduced when they switched over to using `get_response_time_percentile` instead.

This PR simply fixes that regression, switching back to using `get_current_response_time_percentile`.

The bug is easiest to observe if you have a spikey/bursty test shape. During bursts, latency increases. During quieter times, latency goes back down. This is properly reflected in the Locust Web UI's graph tab, but the full history CSV file doesn't show that - instead there's only a slight decline in latency between bursts, because it's showing the 95th percentile since the test began rather than just for the past 10 seconds.

I suspect the bug went just over a year without being noticed because it's unusual to either (1) use a custom load shape for a test and (2) to closely examine the full history CSV file.

cyberw commented:
Interesting. Can you add a test case? (to make sure it doesnt regress again, and to further explain the issue)

TaylorSMarks commented:
@cyberw - I’ve fixed the formatting issues that flake8 was complaining about and updated an existing unit test to cover this.

With my changes, the unit test passes. Without my changes, the unit tests will fail because the 95th percentile will never decrease.

cyberw commented:
Thanks!
